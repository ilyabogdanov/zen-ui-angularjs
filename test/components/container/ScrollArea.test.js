const div = document.createElement("div");
document.body.appendChild(div);

describe("directive: zen-scroll-area", function() {
    let element, scope;
    beforeEach(angular.mock.module("ZenUI"));
    beforeEach(inject(function($compile, $rootScope) {
        scope = $rootScope.$new();
        element = "<zen-scroll-area></zen-scroll-area>";
        element = $compile(element)(scope);
        scope.$digest();
        angular.element(div).append(element);
    }));
    afterEach(function () {
        while (div.firstChild) {
            div.removeChild(div.firstChild);
        }
    });

    it("should render", function () {
        expect(div.childNodes).toHaveLength(1);
        expect(div.firstChild).toMatchSnapshot();
    });
});
