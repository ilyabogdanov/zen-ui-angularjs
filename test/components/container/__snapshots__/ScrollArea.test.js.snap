// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`directive: zen-scroll-area should render 1`] = `
<div
  class="zen_ui__stretch_margin ng-scope ng-isolate-scope"
  ng-style="style"
  ng-transclude=""
  style="overflow-y: auto"
/>
`;
